### DISCUSSION TITLE
after I click "process my application", the page either remain with the button as was not clicked, or just refreshed to the original page

### DISCUSSION THREAD

**Discussion by vxgu86** [community/community#123882] [GitHub Education]
### Hi everyone,

after I click "process my application", the page either remain with the button as was not clicked, or just refreshed to the original page.

what is the problem？ I have tried tons of times, are you really processing?

**Comment by GioPics** [comment]
hi, I had the same problem this morning, after compressing the image it was resolved (from ~700 kb to ~150)
